src/main/java/org/terasology/launcher/util/Platform.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.launcher.util;
+
+public final class Platform {
+
+    private static Platform PLATFORM;
+
+    private String os;
+    private String arch;
+
+    private Platform() {
+        // TODO: consider using regex
+        final String platformOs = System.getProperty("os.name").toLowerCase();
+        if (platformOs.equals("linux")) {
```

Comment:
Are we sure this alwyas _equals_ "linux" or would be using `startsWith` a safer option?

Reply:
rather sure it equals, but probably `startsWith` is a safer option (for the future).